src/integrationTest/resources/caseworker-issue-application-about-to-submit-app2-sol-rep-response.json

```diff
@@ -1,83 +1,106 @@
+    "solServiceMethod": "courtService",
+    "applicant1PrayerHasBeenGivenCheckbox": ["Yes"],
     "applicant1StatementOfTruth": "Yes",
     "solSignStatementOfTruth": "Yes",
     "applicationFeeOrderSummary": {
       "PaymentTotal": "55000"
     },
-    "documentsGenerated": [
-      {
-        "id": "Respondent Invitation",
-        "value": {
-          "documentFileName": "aosinvitation-1616591401473378-2021-06-17:13:00.pdf",
-          "documentType": "aos",
-          "documentLink": {
-            "document_url": "http://dm-store-aat.service.core-compute-aat.internal/documents/c35b1868-e397-457a-aa67-ac1422bb8100",
-            "document_filename": "aosinvitation-1616591401473378-2021-06-17:13:00.pdf",
-            "document_binary_url": "http://dm-store-aat.service.core-compute-aat.internal/documents/c35b1868-e397-457a-aa67-ac1422bb8100/binary"
-          }
-        }
-      },
-      {
-        "id": "Divorce application",
-        "value": {
-          "documentFileName": "divorce-application-1616591401473378-2021-06-17:13:00.pdf",
-          "documentType": "application",
-          "documentLink": {
-            "document_url": "http://dm-store-aat.service.core-compute-aat.internal/documents/5cd725e8-f053-4493-9cbe-bb69d1905ae3",
-            "document_filename": "divorce-application-1616591401473378-2021-06-17:13:00.pdf",
-            "document_binary_url": "http://dm-store-aat.service.core-compute-aat.internal/documents/5cd725e8-f053-4493-9cbe-bb69d1905ae3/binary"
-          }
-        }
-      }
-    ],
-    "applicationPayments": [
-      {
-        "value": {
-          "feeCode": "FEE0001",
-          "amount": 55000,
-          "status": "success",
-          "channel": "online",
-          "reference": "paymentRef",
-          "transactionId": "ge7po9h5bhbtbd466424src9tk"
-        }
-      }
-    ],
     "documentUploadComplete": "Yes",
     "issueDate": "2021-06-17",
-    "dueDate": "2021-07-03"
+    "applicationPayments": [{
+      "value": {
+        "feeCode": "FEE0001",
+        "amount": 55000,
+        "status": "success",
+        "channel": "online",
+        "reference": "paymentRef",
+        "transactionId": "ge7po9h5bhbtbd466424src9tk"
+      }
+    }],
+    "applicant2InviteEmailAddress": "applicant2@test.com",
+    "documentsGenerated": [{
+      "id": "Divorce application",
+      "value": {
+        "documentLink": {
+          "document_url": "http://dm-store-aat.service.core-compute-aat.internal/documents/5cd725e8-f053-4493-9cbe-bb69d1905ae3",
+          "document_filename": "divorce-application-1616591401473378-2021-06-17:13:00.pdf",
+          "document_binary_url": "http://dm-store-aat.service.core-compute-aat.internal/documents/5cd725e8-f053-4493-9cbe-bb69d1905ae3/binary"
+        },
+        "documentFileName": "divorce-application-1616591401473378-2021-06-17:13:00.pdf",
+        "documentType": "application"
+      }
+    }, {
+      "id": "Divorce application",
```

Comment:
id says `Divorce application` but document type is aos 
looks like the order in which document id mock is returned and it is being generated is different